### PR TITLE
Mongolian translation

### PR DESCRIPTION
**Pull request by TuruuDo**
Hey, I happened to play Flexbox Froggy, which a web development online course recommended. My mother tongue is Mongolian, so I decided to add a translation. Please, accept it.

### CODE REVIEW

File: js/levels.js [hunk 1]
```diff
@@ -656,6 +668,7 @@ var levels = [
       'fi': '<p>Auta sammakoita löytämään omat lumpeenlehtensä ominaisuuksien <code>flex-direction</code>, <code>justify-content</code> ja <code>align-items</code> avulla.</p>',
       'gl': '<p>Axuda ás rás a chegar a cadanseu nenúfar empregando <code>flex-direction</code>, <code>justify-content</code>, e <code>align-items</code>.</p>',
       'th': '<p>ช่วยเจ้ากบทั้งหลายไปสู่ใบบัวของมัน ด้วยคำสั่ง <code>flex-direction</code>, <code>justify-content</code>, และ <code>align-items</code>.</p>',
+      'mn': '<p>Мэлхийнүүдэд <code>flex-direction</code>, <code>justify-content</code>, <code>align-items</code>-г ашиглан сараана цэцэгээ олоход нь туслаарай.< /p>'
```
Comment (thomaspark) [inline js/levels.js:671]: `< /p>` should be `</p>`

**Comment by thomaspark** [comment]
Hey @TuruuDo, thanks for the translation. There's one small correction to be made with the `p` tag on `levels.js` line 671. Otherwise it looks ready to merge.

**Comment by TuruuDo** [comment]
Hey, I removed the redundant white space in < /p> on levels.js line 671. Is it ready to merge now?

**Comment by thomaspark** [comment]
Looks good, thanks!

**Pull request merged**